wp-content/plugins/wds-citytech/includes/group-blogs.php

```diff
@@ -347,22 +347,22 @@ function openlab_group_blog_activity( $activity ) {
 
 	$public = get_blog_option( $blog_id, 'blog_public' );
 
-	if ( 0 > (float) $public ) {
+	if ( 0 > (int) $public || 'public' !== $group->status ) {
```

Comment:
What case does this cover? IIRC it's possible to have a group that is private connected to a blog that is public; this line looks like it would force the item to be `hide_sitewide=1` in that situation.

Reply:
Correct. While there's a group to blog connection, their status/privacy settings aren't connected. So you can have a private group with a public site and vice versa.

Reply:
Gotcha. I think that means that we should be more fine-grained. For blog-related activity types, use `blog_public` - I think this is just `new_blog_post`, `new_blog_comment`, and `new_blog`. Otherwise group privacy setting. Does that seem right to you?

Reply:
Make sense, if we don't want to be too strict with `hide_sitewide`. Will push changes if new minutes.